iothub/service/src/ExportImportRequest.cs

```diff
@@ -8,11 +8,7 @@ namespace Microsoft.Azure.Devices
     /// </summary>
     public sealed class ExportImportRequest
     {
-        private string storageConnectionString;
-
-        private string containerName;
-
-        public string StorageConnectionString { get => storageConnectionString; set => storageConnectionString = value; }
-        public string ContainerName { get => containerName; set => containerName = value; }
+        public string StorageConnectionString { get; set; }
+        public string ContainerName { get; set; }
```

Comment:
Does this need to be public? I thought we are using RBAC for storage? is this for non identity path?

Reply:
I didn't change the visibility. We shouldn't change it now that it has been public.

This class is unrelated to the import/export job, but I looked at it because of the name. Then I saw an opportunity to use auto-properties, and cleaned it up.